packages/superset-ui-demo/storybook/shared/components/ResizablePanel.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren, ReactNode } from 'react';
+import { ResizableBox, ResizableBoxProps } from 'react-resizable';
+
+import 'react-resizable/css/styles.css';
+
+export default function ResiablePanel({
```

Comment:
small nit: You have a typo right here @ktmud 

`ResiablePanel` -> `ResizablePanel`